pkg/action/status.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright The Helm Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package action
+
+import (
+	"helm.sh/helm/v3/pkg/release"
+)
+
+// Status is the action for checking the deployment status of releases.
+//
+// It provides the implementation of 'helm status'.
+type Status struct {
```

Comment:
The Status type could be a composite type of helm/pkg/status.Status, with cfg overloaded to our own Configuration type. That way new fields of helm/pkg/status.Status besides Version and ShowDescription would appear for us automatically. And we could reuse helm/pkg/status.Run().

But maybe for this simple type is more hassle than anything.

Reply:
Nice idea, I have no idea how to do that but seems a good opportunity to learn!

Reply:
done, thanks for the hint!